src/lumigo_tracer/spans_container.py

```diff
@@ -115,10 +115,10 @@ def start(self, event=None, context=None):
     def handle_timeout(self, *args):
         get_logger().info("The tracer reached the end of the timeout timer")
         to_send = [s for s in self.spans if s["id"] in self.span_ids_to_send]
+        self.span_ids_to_send.clear()
```

Comment:
this fixes the first bug: the last HTTP call looks infinite although it has finished.
It happens because this call finishes while we send the TO report. I.e. we clean the `self.span_ids_to_send` too late.